src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs

```diff
@@ -76,6 +76,12 @@ public class DownloadPipelineArtifactTaskV1 : PipelineArtifactTaskPluginBaseV1
             string tags = context.GetInput(ArtifactEventProperties.Tags, required: false);
             string userSpecifiedpipelineId = context.GetInput(PipelineId, required: false);
 
+            if (!Directory.Exists(targetPath) && !File.Exists(targetPath))
```

Comment:
I don't think we need this change, if we merge this PR before : https://github.com/microsoft/azure-pipelines-agent/pull/2279/files